recipes/lazydocker/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "lazydocker" %}
+{% set version = "0.23.3" %}
+{% set author = "jesseduffield" %}
```

Comment:
This is only used once, please don't use Jinja for it.
```suggestion
```

Reply:
This is used more than once.

https://github.com/conda-forge/staged-recipes/blob/baa18809804d28eb9fb82c69be76574e739fcabf/recipes/lazydocker/meta.yaml#L25-L33